plugins/catalog-import/src/components/useImportState.ts

```diff
@@ -32,6 +32,12 @@ export type ImportFlows =
 // the available states of the stepper
 type ImportStateTypes = 'analyze' | 'prepare' | 'review' | 'finish';
 
+export type NewLocations = Array<{
```

Comment:
This ends up getting exported all the way out to the public API, I wonder if it just makes sense to keep it inlined for now rather than maintaining another type there?

Reply:
Sure!